dev/IconSource/IconSource.idl

```diff
@@ -90,4 +90,15 @@ unsealed runtimeclass PathIconSource : IconSource
     static Windows.UI.Xaml.DependencyProperty DataProperty { get; };
 }
 
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+unsealed runtimeclass ImageIconSource : IconSource
+{
+    ImageIconSource();
+
+    Windows.UI.Xaml.Media.ImageSource ImageSource{ get; set; };
+
```

Comment:
Since we have the BitmapIconSource type defined here and we can change it, should we just be adding these properties on there? The only reason we're not enhancing BitmapIcon in the OS was because of the OS APIs not going downlevel.

Reply:
Interesting so if BitmapIconSource's ImageSource was not a UriSource what would we do? create an ImageIcon instead?

Reply:
I'm not sure, I actually don't know enough about how that converter function works to say if it would work that way. Or maybe always just create ImageIcon?

Reply:
In that case would/could we delete the ImageIconSource? I think that might be confusing

Reply:
Lets keep ImageIconSource for now and follow up in the spec review.